ci-operator/step-registry/optional-operators/cvp-common/generic-claim/optional-operators-cvp-common-generic-claim-workflow.yaml

```diff
@@ -0,0 +1,12 @@
+workflow:
+  as: optional-operators-cvp-common-generic-claim
```

Comment:
It's no longer a "generic" workflow, could be called just `optional-operators-cvp-common-claim` IMO

Reply:
Renamed.